twisted/test/test_compat.py

```diff
@@ -315,6 +317,26 @@ def test_python3(self):
 
 
 
+class PYPYTest(unittest.SynchronousTestCase):
+    """
+    Identification of PyPy.
+    """
+
+    def test_pypy(self):
+        """
+        On PyPy, C{_PYPY} is True.
```

Comment:
Can you make this L{_PYPY}, so that way it properly hyperlinks in the documentation?